demos/projects/PC/linux/README.md

```diff
@@ -1 +1,103 @@
-# Coming Soon
\ No newline at end of file
+# Use Linux for simulate a device using Azure IoT Middleware for FreeRTOS
+
+## Get the middleware
+
+Clone the following repo to download all sample device code, setup scripts, and offline versions of the documentation. 
+
+**If you previously cloned this repo in another sample, you don't need to do it again.**
+
+```bash
+git clone https://github.com/Azure-Samples/iot-middleware-freertos-samples.git
+```
+
+To initialize the repo, run the following command:
+
+```bash
+cd iot-middleware-freertos-samples
+git submodule update --init --recursive --depth 1
+```
+
+
+## Install Prerequisites
+
+* [CMake](https://cmake.org/download/) (Version 3.20 or higher)
+
+* Execute the installation script 
+
+```bash
+sudo ./.github/scripts/install_software.sh
+```
+
+* Execute the Network setup script 
```

Comment:
Can we say that this script will create veth0 and veth1 interfaces.

Reply:
done!